db/migrate/20180518180406_umm_c_1_9_to_1_10.rb

```diff
@@ -0,0 +1,178 @@
+          coordinate_system = spatial_information['VerticalCoordinateSystem']
+
+          if coordinate_system.key? 'AltitudeSystemDefinition'
+            coordinate_system['AltitudeSystemDefinition'].delete('EncodingMethod')
+          end
+
+          if coordinate_system.key? 'DepthSystemDefinition'
+            coordinate_system['DepthSystemDefinition'].delete('EncodingMethod')
+          end
+        end
+      end
+
+      # Update GeographicCoordinateUnits with enum values
+      # Upcase the name and see if it matches the enum
+      geographic_coordinate_units = draft.fetch('SpatialInformation', {}).fetch('HorizontalCoordinateSystem', {}).fetch('GeographicCoordinateSystem', {}).delete('GeographicCoordinateUnits')
+      unless geographic_coordinate_units.nil?
+        new_unit = nil
+        ['Decimal Degrees', 'Kilometers', 'Meters'].each do |unit|
+          if unit.upcase == geographic_coordinate_units.upcase
+            new_unit = unit
+          end
+        end
+
+        draft['SpatialInformation']['HorizontalCoordinateSystem']['GeographicCoordinateSystem']['GeographicCoordinateUnits'] = new_unit unless new_unit.nil?
+      end
+
+      # Update DistanceUnits with enum values
+      # Altitude DistanceUnits
+      altitude_distance_units = draft.fetch('SpatialInformation', {}).fetch('VerticalCoordinateSystem', {}).fetch('AltitudeSystemDefinition', {}).delete('DistanceUnits')
+      unless altitude_distance_units.nil?
+        new_unit = nil
+        %w[HectoPascals Kilometers Millibars].each do |unit|
+          new_unit = unit if unit.upcase == altitude_distance_units.upcase
+        end
+
+        draft['SpatialInformation']['VerticalCoordinateSystem']['AltitudeSystemDefinition']['DistanceUnits'] = new_unit unless new_unit.nil?
+      end
+
+      # Depth DistanceUnits
+      depth_distance_units = draft.fetch('SpatialInformation', {}).fetch('VerticalCoordinateSystem', {}).fetch('DepthSystemDefinition', {}).delete('DistanceUnits')
+      unless depth_distance_units.nil?
+        new_unit = nil
+        %w[Fathoms Feet HectoPascals Meters Millibars].each do |unit|
+          new_unit = unit if unit.upcase == depth_distance_units.upcase
+        end
+
+        draft['SpatialInformation']['VerticalCoordinateSystem']['DepthSystemDefinition']['DistanceUnits'] = new_unit unless new_unit.nil?
+      end
+
+      # util/remove-empty-maps
+
+      # (update-in [:SpatialExtent :VerticalSpatialDomains] spatial-conversion/drop-invalid-vertical-spatial-domains)
+      # "Any VerticalSpatialDomain with a Type not in valid-vertical-spatial-domain-types
+      # is not considered valid according to UMM spec v1.10.0 and should be dropped.
+      # This behavior will eventually generate errors"
+
+      # I don't think we should completely delete the VerticalSpatialDomain, we can just remove the Type field, schema validation will point out the missing field
+      # TODO look at invalid picklists instead of deleting the field
```

Comment:
remove TODO

don't delete the value if it doesn't match the new enum, just let it be invalid